spellbook/tests/transfers_ethereum/transfers_ethereum_erc1155_assert_samples_single.sql

```diff
@@ -0,0 +1,13 @@
+-- Bootstrapped correctness test against legacy Postgres values.
+-- Also manually check etherscan info for the first 5 rows
```

Comment:
we could make that manual check into a actual test but we should have enough coverage using the legacy vals.